### PR TITLE
add static assets and uxpl for rendering default programs certificate

### PR DESCRIPTION
**Pull request by zubair-arbi**
ECOM-3027
@awais786 @ahsan-ul-haq @tasawernawaz 
* add `edx-pattern-library` with version `0.8.5`
* add configuration for js files of `edx-pattern-library`
* add basic sass for web certificates from [web certificates repo](https://github.com/edx/web-certificates/tree/master/static/sass)
* add necessary images from certificate rendering from [web certificates repo](https://github.com/edx/web-certificates/tree/master/static/images)

**Note:**
_The sass files from web certificates repo still needs some changes and we will add those changes in a separate PR in order exclude wait time for merging this PR.
https://github.com/edx/web-certificates/pull/35_

_Old PR (Closed): https://github.com/edx/credentials/pull/32_

### CODE REVIEW

**Comment by zubair-arbi** [comment]
@clintonb @AlasdairSwan Please review.
FYI: There are [some more changes](https://github.com/edx/web-certificates/pull/35) which are in flight by @marcotuts, we will add those changes separately.

File: credentials/static/sass/_config.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// ------------------------------
+// edX Web Certificates: Config
+
+// About: variable and configuration overrides
+
+// ------------------------------
+// #VARIABLES
+// ------------------------------
+ $font-path: '../bower_components/edx-pattern-library/pattern-library/fonts'; // LOCALDEV
```
Comment (clintonb) [inline credentials/static/sass/_config.scss]: Get rid of all of the dead code, and LOCALDEV comments.

File: credentials/static/sass/mitpe/_utilities.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+// #VARIABLES
+// #FUNCTIONS
+// #MIXINS
+// #HELPERS
+
+// ------------------------------
+// #VARIABLES
+// ------------------------------
+ $font-path: '/node_modules/edx-pattern-library/pattern-library/fonts';
+// $font-path: '/media/certificate_template_assets/'; // CERTPROD
```
Comment (clintonb) [inline credentials/static/sass/mitpe/_utilities.scss]: Remove.

File: credentials/static/sass/mitpe/_utilities.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+// #VARIABLES
+// #FUNCTIONS
+// #MIXINS
+// #HELPERS
+
+// ------------------------------
+// #VARIABLES
+// ------------------------------
+ $font-path: '/node_modules/edx-pattern-library/pattern-library/fonts';
```
Comment (clintonb) [inline credentials/static/sass/mitpe/_utilities.scss]: This should come from bower.

File: credentials/templates/base.html [hunk 4]
```diff
@@ -12,7 +12,7 @@
     <title>{% block title %}{% endblock title %}</title>
 
     {% compress css %}
-        <link rel="stylesheet" href="{% static 'sass/main.scss' %}" type="text/x-scss">
+        <link rel="stylesheet" href="{% static 'sass/main-ltr.scss' %}" type="text/x-scss">
```
Comment (clintonb) [inline credentials/templates/base.html]: What about rtl?

Comment (zubair-arbi) [inline credentials/templates/base.html]: According to discussion with @marcotuts we will support only `ltr` for now until we have a proper styling for `rtl`.

**Comment by zubair-arbi** [comment]
@clintonb Please review.
@marcotuts @peter-fogg it would be great if someone of you can review too.

**Comment by clintonb** [comment]
None of the images in `/static/images/dev/` are used. If we aren't using them, they should not be in the repo. If they will be used later, add them in that PR.

**Comment by clintonb** [comment]
All of the MIT PE SASS should be removed until we actually need it.

**Comment by clintonb** [comment]
:+1: (once unused files removed)

**Comment by marcotuts** [comment]
@zubair-arbi - my changes merged into web-certificates.

**Comment by peter-fogg** [comment]
I agree about removing dead code; otherwise :+1:

**Comment by zubair-arbi** [comment]
@clintonb I have removed unused images and scss files.
@marcotuts I have update the scss files with your new changes from `web-certificates` repo.

**Pull request merged**